uavcan/time/SynchronizedTimestamp.1.0.uavcan

```diff
@@ -1,16 +1,14 @@
 #
-# Nested data type used for representing a point in time with microsecond resolution.
+# Nested data type used for representing a network-wide synchronized timestamp with microsecond resolution.
 # This data type is highly recommended for use both in standard and vendor-specific messages alike.
 #
 
 # Zero means that the time is not known.
 uint56 UNKNOWN = 0
```

Comment:
You know what I think of these "Special null values", but I think it might be acceptable here to get the same amount of bits before time sync is achieved.